src/components/Apps/VariableCreateUpdateForm/index.vue

```diff
@@ -25,7 +25,7 @@ export default {
   data() {
     return {
       submitBtnText: this.$t('Confirm'),
-      url: '/api/v1/ops/variable/',
+      url: '/api/v1/ops/variables/',
       form: Object.assign({ 'on_invalid': 'error' }, this.variable || {}),
       fields: [
         ['', ['name', 'var_name', 'type', 'text_default_value', 'select_default_value', 'extra_args', 'tips', 'required']]
```

Comment:
There is no specific query request to focus on these two changes. Please provide more information regarding which sections of the code you would like to discuss or address irregularities, potential issues, and optimizations suggestions.